src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2212,10 +2212,10 @@ properties:
                 type: string
                 fits_keyword: R_V2V3
                 blend_table: True
-          undersampling_correction:
-            title: Undersampling Correction
+          charge_migration:
```

Comment:
1) This whole entry should be moved down in to the "cal_step" section of the header, with all the other "S_xxxxx" keywords. This is the section for reference file names.  2) The title should be "Charge Migration" (mixed case). 3) the keyword name should be "S_CHGMIG", which is more consistent with the step name.

Reply:
Done